src/main/java/us/ihmc/handsros2/abilityHand/AbilityHandManager.java

```diff
@@ -120,12 +125,14 @@ public float getStageFingerPosition(int stage, int finger)
       }
    }
 
-   private static final float TOLERANCE = 2.0f;
-   private static final float THUMB_CLEAR_POSITION = 30.0f;
+   private static final float TOLERANCE = 1.0f;
```

Comment:
I reduced the tolerance. Controllers are inactive within a +/- 1 degree, so 2 degree deadband